### ISSUE TITLE
Post Processing unable to copy file/Unable to move the files to their new home

### ISSUE THREAD

**Issue by phoenix7700** [SickChill/SickChill#6921]
I think the problem is related to permissions but I gave 777 permissions to both the destination and source but the error still persists.

### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit:
OS: TrueNAS-12.0-RELEASE
Browser: Chrome Version 87.0.4280.88
What you did: Manual pre-processing
What happened: Post processing didn't copy the files to the new destination
What you expected: For the files to be copies to the new location
Logs:
```
2020-12-05 13:11:19 INFO :: POSTPROCESSOR-MANUAL :: Manual post processing task for /Shared/Downloading/SEAL.Team.S04E01.720p.WEB.H264-CAKES[rarbg] completed
2020-12-05 13:11:19 INFO :: POSTPROCESSOR-MANUAL :: /Shared/Downloading/SEAL.Team.S04E01.720p.WEB.H264-CAKES[rarbg]/seal.team.s04e01.720p.web.h264-cakes.mkv : Processing failed: Unable to move the files to their new home
2020-12-05 13:11:19 INFO :: POSTPROCESSOR-MANUAL :: Some items were not processed.
2020-12-05 13:11:19 WARNING :: POSTPROCESSOR-MANUAL :: Processing Failed
2020-12-05 13:11:19 WARNING :: POSTPROCESSOR-MANUAL :: Processing failed for /Shared/Downloading/SEAL.Team.S04E01.720p.WEB.H264-CAKES[rarbg]/seal.team.s04e01.720p.web.h264-cakes.mkv: Unable to move the files to their new home
2020-12-05 13:11:19 ERROR :: POSTPROCESSOR-MANUAL :: Unable to copy file /Shared/Downloading/SEAL.Team.S04E01.720p.WEB.H264-CAKES[rarbg]/seal.team.s04e01.720p.web.h264-cakes.mkv to /Shared/TV/SEAL Team/Season 04/SEAL.Team.S04E01.720p.WEB.DL-cakes.mkv: [Errno 2] No such file or directory: '/Shared/TV/SEAL Team/Season 04/SEAL.Team.S04E01.720p.WEB.DL-cakes.mkv'
2020-12-05 13:11:19 INFO :: POSTPROCESSOR-MANUAL :: Release name (seal.team.s04e01.720p.web.h264-cakes) found from file (seal.team.s04e01.720p.web.h264-cakes.nfo)
2020-12-05 13:11:19 INFO :: POSTPROCESSOR-MANUAL :: Processing /Shared/Downloading/SEAL.Team.S04E01.720p.WEB.H264-CAKES[rarbg]/seal.team.s04e01.720p.web.h264-cakes.mkv (None)
2020-12-05 13:11:19 INFO :: POSTPROCESSOR-MANUAL :: Processing /Shared/Downloading/SEAL.Team.S04E01.720p.WEB.H264-CAKES[rarbg]
2020-12-05 13:11:19 INFO :: POSTPROCESSOR-MANUAL :: Beginning manual post processing task: /Shared/Downloading/SEAL.Team.S04E01.720p.WEB.H264-CAKES[rarbg]
2020-12-05 13:11:19 INFO :: WEBSERVER-POSTPROCESS_0 :: Manual post processing task for /Shared/Downloading/SEAL.Team.S04E01.720p.WEB.H264-CAKES[rarbg] was added to the queue
```

**Comment by miigotu** [comment]
Looks like the show directory doesn't exist

**Comment by phoenix7700** [comment]
![issue](https://user-images.githubusercontent.com/6169898/101272185-864bf100-374f-11eb-9c91-0efd91a5b422.PNG

Not sure how to give anymore proof that the directory exists.

**Comment by phoenix7700** [comment]
Made some progress on this. Was able to get post processing to work if I set permissions of both the source and destination folders to the UID of the sickchill user which is 1001 in my case, but this isn't what I want because it messes up the other permissions I have set. I tried changing the UID of sickchill inside the iocage to match the permissions that I have for the folder but that cause the plugin to not load the web interface at all.
